arch/arm/soc/nxp_imx/rt/soc.c

```diff
@@ -30,74 +28,6 @@ const clock_usb_pll_config_t usb1PllConfig = {
 	.loopDivider = 0U
 };
 
-static void BOARD_BootClockGate(void)
```

Comment:
Is it ok that this isn't moved to the board code? ie that we aren't setting these registers up at all?